src/schema-templates/recipe.block.php

```diff
@@ -23,8 +24,8 @@
 ];
 
 $yoast_seo_recommended_blocks = [
-	[ 'name' => 'core/paragraph' ],
 	[ 'name' => 'yoast/cooking-time' ],
```

Comment:
It's not technically part of this PR, but should `yoast/cooking-time` not be listed under `$yoast_seo_block_template` as well?